ui/src/personal/Personal.js

```diff
@@ -80,6 +81,10 @@ const SelectTheme = (props) => {
   const themeChoices = Object.keys(themes).map((key) => {
     return { id: key, name: themes[key].themeName }
   })
+  themeChoices.push({
+    id: autoKey,
+    name: 'Auto',
```

Comment:
Can you please add "Auto" as the first option? Maybe even making it the default option? What do you think @jvoisin ?

Reply:
Done. Added Auto as the first option.
I think we should not default to light unless explicitly said by user, nobody 'dislikes' dark theme, but many 'hate' light. Just a personal opinion though 😛

Reply:
The default is dark. If it is light on your environment, maybe there's an issue with your implementation. After you work on the last changes I pointed out, I'll do a thoroughly test.

Reply:
Oh, it is dark on my environment too. I meant to say if we have an 'auto' as the default, users with a light theme in the OS will get a default light theme.

Reply:
We are keeping "Dark" as the default theme for now. I thought about using auto as the default, but I'm not sure about it. Once we have this merged, I may ask to our users for their opinion before the next release.

Reply:
Just to be clear: Being the first option in the menu does not make it the default. At least it should not...

Reply:
No, the default is still dark. Being the first option doesn't affect it.